aequilibrae/paths/traffic_assignment.py

```diff
@@ -295,6 +299,37 @@ def set_cores(self, cores: int) -> None:
             c.results.set_cores(cores)
             c._aon_results.set_cores(cores)
 
+    def set_save_path_files(self, save_it: bool) -> None:
```

Comment:
What if we add a class after setting it to save path?

Reply:
Maybe a list of traffic classes should be in the instantiation of the TrafficAssignment class?

Reply:
it's really a convenience method that applies it to a list of current classes. either you add them on instantiation and then make them fixed, or you keep a flag that then applies this to each class you add.

Reply:
that's really a user question so I am happy with any consistent definition as long as we make it specific